apps/dashboard/lib/auth/workos.ts

```diff
@@ -290,11 +335,12 @@ export class WorkOSAuthProvider extends BaseAuthProvider {
       expiresAt.setDate(expiresAt.getDate() + 7);
 
       return {
-        newToken: refreshResult.sealedSession!,
+        sessionToken: refreshResult.sealedSession!,
```

Comment:
I am against the non-null assertion operator.... however this is up to you as you will know if there is any possibility in this context that we would be null. 

I'd prefer if we type narrow
```
let sessionToken = null;
if (refreshResult.sealedSession) {
  // safe to access refreshResult.sealedSession here
  sessionToken = refreshResult.sealedSession
} else {
  // Handle the case where sealedSession is null or undefined
  console.warn("Sealed session is missing!");
}

return {...}
````

Reply:
It would only be null if authenticated was false, which there's a check for above, but I guess typescript doesn't know that. I added additional guards/checks with a comment that if we ever got this error, its because WorkOS messed up, not us.